### PR TITLE
Use a rake task to obtain private key and cert

### PR DESCRIPTION
**Pull request by adamruzicka**
This also gives us free access to RH account numbers, but requires to be run against Satellite >=6.8

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -45,23 +45,25 @@
...
-      force_basic_auth: yes
-    register: sat_orgs_json
-  - name: Extract organization configuration per account and save to disk
-    include_tasks: extract_orgs.yml
-    loop: "{{ sat_orgs_json.json | json_query('results[*].id') }}"
-    when: not org_id in skip_satellite_org_id_list
-    loop_control:
-      loop_var: org_id
+    - name: Dump organization details
+      command: foreman-rake katello:receptor:extract_orgs OUTPUT_FILE=/tmp/receptor.json
```
Comment (ares) [inline tasks/main.yml:51]: what happens if this is ran in parallel for two orgs? should we generate proper tempfile somehow to be on the safe side?

File: tasks/main.yml [hunk 2]
```diff
@@ -45,23 +45,25 @@
...
+    - name: Retrieve dumped organization details
+      slurp:
+        src: /tmp/receptor.json
```
Comment (ares) [inline tasks/main.yml:54]: this would need to be modified too then

File: tasks/main.yml [hunk 3]
```diff
@@ -45,23 +45,25 @@
...
+      register: org_details
+    - name: Remove dumped organization details
+      file:
+        path: /tmp/receptor.json
```
Comment (ares) [inline tasks/main.yml:58]: and here

File: tasks/main.yml [hunk 4]
```diff
@@ -45,23 +45,25 @@
...
-      validate_certs: "{{ satellite_validate_certs }}"
-      force_basic_auth: yes
-    register: sat_orgs_json
-  - name: Extract organization configuration per account and save to disk
-    include_tasks: extract_orgs.yml
-    loop: "{{ sat_orgs_json.json | json_query('results[*].id') }}"
-    when: not org_id in skip_satellite_org_id_list
-    loop_control:
-      loop_var: org_id
+    - name: Dump organization details
```
Comment (ares) [inline tasks/main.yml:50]: what permissions this sets for the file? could someone read the content? (race condition)

Comment (adamruzicka) [reply tasks/main.yml:50]: 644, owned by foreman:foreman

File: tasks/main.yml (hunk 1 above)
Comment (adamruzicka) [reply tasks/main.yml:51]: The rake task gives details for all organizations so it is not really being run for an organization. The issue with running this multiple times at once is real though.

File: tasks/main.yml (hunk 4 above)
Comment (adamruzicka) [reply tasks/main.yml:60]: A temporary file is now used. It is still owned by `foreman:foreman`, but has 600 permissions

**Pull request merged**